### PR TITLE
update author field for 2022-11-10-wildfly-jakartaee.adoc since scottmarlow doesn't work.  Try switch to smarlow

### PR DESCRIPTION
**Pull request by scottmarlow**
Signed-off-by: Scott Marlow <smarlow@redhat.com>

### CODE REVIEW

**Comment by bstansberry** [comment]
Hi Scott

You need something for these two things:

https://github.com/wildfly/wildfly.org/tree/main/assets/img/news
https://github.com/wildfly/wildfly.org/blob/main/_data/authors.yaml

What you use in the author tag in the posts needs to map to what you use in authors.yaml. What you use is up to you.

Sorry for not noticing this when I looked at the rendered post.

**Comment by scottmarlow** [comment]
Thanks @bstansberry , I changed this pr to update https://github.com/wildfly/wildfly.org/tree/main/assets/img/news + https://github.com/wildfly/wildfly.org/blob/main/_data/authors.yaml

File: _data/authors.yaml [hunk 1]
```diff
@@ -171,4 +171,11 @@ pferraro:
   occupation: "Principal Software Engineer"
   bio: "https://github.com/pferraro"
   avatar: pferraro.png
+scottmarlow:
+  name:"Scott Marlow"
```
Comment (bstansberry) [inline _data/authors.yaml:175]: You need a space here after the colon.

Comment (scottmarlow) [reply _data/authors.yaml]: thanks, space added!

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @scottmarlow

**Pull request merged**